tasks/giovanni-adapter/app/stac/types.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * A link within a STAC item or catalog
+ * https://github.com/radiantearth/stac-spec/blob/master/catalog-spec/catalog-spec.md#link-object
+ * https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md#link-object
+ */
+export interface StacLink {
+  rel: 'self' | 'root' | 'parent' | 'child' | 'item' | 'collection' | 'derived_from' | 'alternate' | string;
```

Comment:
I think this can just be `rel: string` since it won't enforce the other entries since they are a subset of `string`

Reply:
Good point:)
But since Chris is suggesting moving the stac stuff in a separate library in the future, do you think we should defer this change until then?

Reply:
Got it ... fixed.